omod/src/main/webapp/resources/htmlforms/Covid19-ReferralForm.xml

```diff
@@ -0,0 +1,376 @@
+                                        <div class="row">
+                                            <div class="col-md-3">
+                                                <label>Referring Facility</label>
+                                                <obs conceptId="159395" />
+                                            </div>
+                                            <div class="col-md-3">
+                                                <label>Ward/Unit</label>
+                                                <obs conceptId="159395" />
+                                            </div>
+                                            <div class="col-md-3">
+                                                <label>Referring Clinician</label>
+                                                <encounterProviderAndRole encounterRole="240b26f9-dd88-4172-823d-4a8bfeb7841f" required="required" />
+                                            </div>
+                                            <div class="col-md-3">
+                                                <label>Phone</label>
+                                                <obs conceptId="159395" />
+                                            </div>
+                                        </div>
+                                        <div class="row">
+                                            <div class="col-md-3">
+                                                <label>Designation</label>
+                                                <obs conceptId="159395" />
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+
+                                <div class="card">
+                                    <div class="card-header">
+                                            Recieving Facility
+                                        </div>
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-3">
+                                                <label>Recieving Facility</label>
+                                                <obs conceptId="159395" />
+                                            </div>
+                                            <div class="col-md-3">
+                                                <label>Ward/Unit</label>
+                                                <obs conceptId="159395" />
+                                            </div>
+                                            <div class="col-md-3">
+                                                <label>Recieving Clinician</label>
+                                                <encounterProviderAndRole encounterRole="240b26f9-dd88-4172-823d-4a8bfeb7841f" required="required" />
```

Comment:
How do we differentiate  between a Medical Officer. Looks to me they share the same enounterRole, which makes it difficult to separate that data if we needed to report about medical officers and referring clinician.

Reply:
I think the best way is to wrap them under grouping concepts